dashboard/test/models/script_test.rb

```diff
@@ -1308,8 +1309,8 @@ class SummarizeVisibleAfterScriptTests < ActiveSupport::TestCase
     create :script_level, script: script, lesson: lesson3, bonus: true
     create :script_level, script: script, lesson: lesson3, bonus: true
 
-    bonus_levels1 = script.get_bonus_script_levels(lesson1)
-    bonus_levels3 = script.get_bonus_script_levels(lesson3)
+    bonus_levels1 = script.get_bonus_script_levels(lesson1, student)
+    bonus_levels3 = script.get_bonus_script_levels(lesson3, student)
```

Comment:
the method takes a user_id as a param, but you're passing a whole user here; does it need to be student.id?

Reply:
get_bonus_script_levels takes the whole user object -> summarize_as_bonus just takes the user id

Reply:
ah - got it. thanks!